spec.md

```diff
@@ -375,14 +444,15 @@ set, the last (greatest) element is the furthest away.
 
 ### Bucket Index
 
-The bucket index can be computed using the following function:
+The index of bucket can be computed using the following function:
```

Comment:
Why? Also, this should now be "The index of the bucket", but then this is exactly the same as "The bucket index", so why the change?